rules/TypeDeclaration/Rector/FunctionLike/AddClosureParamTypeFromArgClassStringRector.php

```diff
@@ -69,11 +67,12 @@ public function getNodeTypes(): array
     }
 
     /**
-     * @param CallLike $node
+     * @param MethodCall|StaticCall $node
```

Comment:
It's always better to have exact same types as in `getNodeTypes()`. The `CallLike` is to generic and it gives the `refactor()` method types that are never passed.